cmd/titus-vpc-tool/setup_container.go

```diff
@@ -20,10 +18,7 @@ func setupContainercommand(ctx context.Context, v *pkgviper.Viper, iipGetter ins
 			netns := v.GetInt("netns")
 			bandwidth := v.GetInt64("bandwidth")
 			burst := v.GetBool("burst")
-			jumbo := v.GetBool("jumbo")
 			switch strings.ToLower(v.GetString(generationFlagName)) {
-			case "v1":
-				return container.SetupContainer(ctx, iipGetter(), netns, uint64(bandwidth), burst, jumbo)
 			case "v2", "v3":
```

Comment:
remove v2 also if you want to get rid of both?

Reply:
V2 was already partially removed.